resources/authgear/templates/__default__/translation.json

```diff
@@ -88,15 +88,15 @@
   "oob-otp-page-title--sms": "SMS One-Time-Password",
   "oob-otp-page-title--email": "Email One-Time-Password",
   "oob-otp-placeholder": "code",
-  "oob-otp-description": "We have sent a {length} digit code to {target}. Please enter the code below to continue",
+  "oob-otp-description": "We have sent a {length} character code to {target}. Please enter the code below to continue",
```

Comment:
Would it be more grammatically correct if we add a hyphen between `{length}` and `character`?

Reply:
adding hyphen between length and character sounds better, updated and rebased